control-plane/api-gateway/consul/config_entry_translation.go

```diff
@@ -0,0 +1,94 @@
+package consul
+
+import (
+	"os"
+	"strings"
+
+	gwv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
+
+	"github.com/hashicorp/consul-k8s/control-plane/namespaces"
+	"github.com/hashicorp/consul/api"
+	consulAPI "github.com/hashicorp/consul/api"
+)
+
+const (
+	metaKeyManagedBy       = "managed-by"
+	metaValueManagedBy     = "consul-k8s-gateway-controller"
+	metaKeyKubeNS          = "k8s-namespace"
+	metaKeyKubeServiceName = "k8s-service-name"
+
+	AnnotationGateway = "consul.hashicorp.com/gateway"
+)
+
+type consulIdentifier struct {
+	name      string
+	namespace string
+	partition string
+}
+
+type NamespaceName struct {
+	Namespace string
+	Name      string
+}
```

Comment:
should be able to use `types.NamespacedName` from `"k8s.io/apimachinery/pkg/types"`

Reply:
oh good call, I was looking for a `types` package in this codebase